### ISSUE TITLE
New model creation

### ISSUE THREAD

**Issue by jjkoehorst** [opencobra/memote#455]
When creating a new model using the template option it is fixed to GitHub.

```
github_username [authorname]: 
project_name [Memote model repository]: 
```

Would it be possible to have a question before or integrated where you can specify the git url
For example I mostly work in gitlab but we also have group projects in which I could want to build a model in. For example a url could be: 

http://gitlab.com/wurssb/memote-demo/

or

http://mypersonalgitwebsite.com/private/memote-demo/

I can change the git settings using

git remote set-url origin ..... but it would be nice if this could be set at init stage?

**Comment by Midnighter** [comment]
Yes, what do you think is better: 1) the option to choose between a few pre-generated options (GitHub, Bitbucket, Gitlab) or 2) a free field?

**Comment by jjkoehorst** [comment]
A free field is very important in the sense that varying universities and research centers have their own flavour of gitlab running. However, as most modellers are not that into git I can assume that for those it would be easiest for a username / repo name in GitHub (which currently is already setup).

so I would suggest:

Which git flavour would you like to use: bitbucket / gitlab / GitHub / other (default GitHub)

if GitHub use the current workflow

if bitbucket / gitlab / other ... please give the complete url of an empty remote repository: http://xxx.../project.git

Something like this?
